tests/test_delete_activity.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from app import app
+from flask import url_for
+from urlparse import urlparse
+import pymesync
+
+
+class DeleteActivityTestCase(unittest.TestCase):
+
+    def setUp(self):
+        app.config['TESTING'] = True
+        app.config['WTF_CSRF_ENABLED'] = False
+
+        self.client = app.test_client()
+
+        self.ctx = app.test_request_context()
+        self.ctx.push()
+
+        self.baseurl = app.config['TIMESYNC_URL']
+
+    def tearDown(self):
+        self.ctx.pop()
+
+    def login_admin(self):
```

Comment:
`login_admin` and `login_noadmin` could just be `login(self, username='some_default', password='some_default_password')`. It's fine as is, just a thought. Less code ~= easier to maintain.